src/components/WwwFrame/PromotionalBanner/Banners/GenericPromoBanner.vue

```diff
@@ -1,27 +1,28 @@
 <template>
 	<div class="row align-center generic-banner">
-		<component
-			:is="currentWrapperComponent"
-			:to="trimmedLink"
-			:href="trimmedLink"
-			:target="isExternalLink ? '_blank' : '_self'"
-			:class="{ 'banner-link' : trimmedLink, 'banner-wrapper' : !trimmedLink}"
-			v-kv-track-event="handleTracking"
-		>
-			<kv-icon :name="iconKey" :class="`${iconKey}-icon icon`" />
-			<div class="content" v-html="promoBannerContent.richText">
-			</div>
-		</component>
-		<a
-			v-if="hasDisclaimer"
-			@click="scrollToSection('#disclaimers')"
-			class="disclaimer-indicator"
-			v-kv-track-event="['promo', 'click-Contentful-banner', 'disclaimer-superscript', '1']"
-		>
-			<sup>
-				1
-			</sup>
-		</a>
+		<kv-icon :name="iconKey" :class="`${iconKey}-icon icon`" />
+		<span class="text-center">
+			<component
+				:is="currentWrapperComponent"
+				:to="trimmedLink"
+				:href="trimmedLink"
+				:target="isExternalLink ? '_blank' : '_self'"
+				:class="{ 'banner-link' : trimmedLink, 'banner-wrapper' : !trimmedLink}"
+				v-kv-track-event="handleTracking"
+			>
+				{{ processedContent }}
+			</component>
+			<a
```

Comment:
By stripping out the `p` tags injected by Contentful we can use `processedContent` directly and set these items to be inline instead of one block alongside the other. Previously the `a` tag wrapped around the `p` from contentful created invalid markup which forced a block layout.